src/Platforms/MySQLPlatform.php

```diff
@@ -186,25 +181,16 @@ public function getListViewsSQL($database)
      */
     public function getListTableForeignKeysSQL($table, $database = null)
     {
-        $table = $this->quoteStringLiteral($table);
-
-        if ($database !== null) {
-            $database = $this->quoteStringLiteral($database);
-        }
-
-        $sql = 'SELECT DISTINCT k.`CONSTRAINT_NAME`, k.`COLUMN_NAME`, k.`REFERENCED_TABLE_NAME`, ' .
-               'k.`REFERENCED_COLUMN_NAME`, k.`ORDINAL_POSITION` /*!50116 , c.update_rule, c.delete_rule */ ' .
-               'FROM information_schema.key_column_usage k /*!50116 ' .
-               'INNER JOIN information_schema.referential_constraints c ON ' .
-               '  c.constraint_name = k.constraint_name AND ' .
-               '  c.table_name = ' . $table . ' */ WHERE k.table_name = ' . $table;
-
-        $databaseNameSql = $database ?? 'DATABASE()';
-
-        return $sql . ' AND k.table_schema = ' . $databaseNameSql
-            . ' /*!50116 AND c.constraint_schema = ' . $databaseNameSql . ' */'
-            . ' AND k.`REFERENCED_COLUMN_NAME` is not NULL'
-            . ' ORDER BY k.`ORDINAL_POSITION`';
+        return 'SELECT k.CONSTRAINT_NAME, k.COLUMN_NAME, k.REFERENCED_TABLE_NAME, ' .
+               'k.REFERENCED_COLUMN_NAME /*!50116 , c.UPDATE_RULE, c.DELETE_RULE */ ' .
+               'FROM INFORMATION_SCHEMA.KEY_COLUMN_USAGE k /*!50116 ' .
```

Comment:
We'll need to deprecate and drop the support for MySQL 5.6 and older ([reference](https://www.mysql.com/support/supportedplatforms/database.html)).